File: java/src/main/java/Movie.java
import org.json.JSONObject;

public class Movie {
    Integer id;
    Integer categoryId;
    String title;
    Boolean isNew;

    //JSON serializer constructor
    public Movie(Integer id, Integer categoryId, String title, Boolean isNew) {
        this.id = id;
        this.categoryId = categoryId;
        this.title = title;
        this.isNew = isNew;
    }

    Movie(Integer categoryId, String title, Boolean isNew) {
        this.categoryId = categoryId;
        this.title = title;
        this.isNew = isNew;
    }

    @Override
    public String toString() {
        return new JSONObject(this).toString();
    }
}

File: c_sharp/MockingExample/App/Movie.cs
using System;
using Newtonsoft.Json;

namespace MovieRentalApp
{
    public class Movie
    {
        int id;
        int categoryId;
        String title;
        bool isNew;

        public Movie(string title, int categoryId, bool isNew)
        {
            this.title = title;
            this.categoryId = categoryId;
            this.isNew = isNew;
        }

        public override String ToString()
        {
            return "{title:\""+title+"\", category:"+categoryId+", isNew:"+isNew+"}";
        }
    }
}